sonia_sonar_states/src/sonia_sonar_states/start_bundle.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python 
+#-*- coding: utf-8 -*-
+
+from time import time
+
+import rospy
+
+from flexbe_core import EventState, Logger
+from std_msgs.msg import Bool, String
+
+class start_bundle(EventState):
+
+    '''
+        Create the object for the trajectory
+
```

Comment:
Missing variables descriptions : Obstacle2Search (should be replaced by target to be consistent with vision states) and resetBundle